exe/ejson_wrapper

```diff
@@ -10,7 +10,7 @@ options = {
   kms_key_id: nil
 }
 option_parser = OptionParser.new do |opts|
-  opts.banner = 'Usage: ejson_wrapper generate [options]'
+  opts.banner = 'Usage: ejson_wrapper {generate,decrypt} [options]'
```

Comment:
🤔 Is it a bit strange to have the commands listed there? Normally I see something like this:

```
Usage: ejson_wrapper <command> [<option> ...]

Commands:
  generate ...
  decrypt ...
```

But I don't know how much effort this would be.